Update README.md of aws Cloudwatch-logs integration

Avivsc-corl opened the pull request:
Update README.md of aws Cloudwatch-logs integration

Avivsc-corl committed:
update 1 README.md cloudwatch-logs

Avivsc-corl committed:
update 2 README.md cloudwatch-logs

MichaelBriggs-Coralogix commented:
We'll hold off on approving this PR until you update the README again with the new custom URL support.

Avivsc-corl committed:
Update template.yaml

Add custom url + update ssm to work with sns

Avivsc-corl committed:
Update README.md

Added customDomain

Avivsc-corl committed:
Update README.md

Update visual

Avivsc-corl committed:
Update README.md

update visual

Avivsc-corl committed:
Update README.md

update visual #3

Avivsc-corl committed:
Update README.md

update visual #4

Avivsc-corl committed:
Update README.md

update visual #5

Avivsc-corl committed:
Update template.yaml

Remove unnecessary lines

Avivsc-corl committed:
Update package.json

update version number

Avivsc-corl committed:
Update README.md

Change the first line, edit private-key guidance and change the link to aws cloudwatch logs.

Avivsc-corl committed:
Update README.md

update link to Coralogix-Lambda-SSMLayer

Avivsc-corl committed:
Update template.yaml

change version

Avivsc-corl committed:
Update README.md

Update link and lines

Avivsc-corl committed:
Update README.md

update

Avivsc-corl committed:
Update README.md

add appname/subsystemname from the json log
